CONTRIBUTING.md

```diff
@@ -199,51 +208,22 @@ Good pull requests — patches, improvements and new features — are a fantasti
 
 When making a contribution, adhering to the [GitHub flow](https://guides.github.com/introduction/flow/index.html) process is the best way to get your work merged:
 
-1. [Fork](http://help.github.com/fork-a-repo/) the repo, clone your fork and configure the remotes:
-
-   ```bash
-   # Clone your fork of the repo into the current directory.
-   git clone https://github.com/<your-username>/activist.git
-   # Navigate to the newly cloned directory.
-   cd activist
-   # Assign the original repo to a remote called "upstream".
-   git remote add upstream https://github.com/activist-org/activist.git
-   ```
-
-   Now, if you run `git remote -v` you should see two remote repositories named:
-
-   - `origin` (forked repository)
-   - `upstream` (activist repository)
-
-2. If you cloned a while ago, get the latest changes from upstream:
+1. If you cloned a while ago, get the latest changes from upstream:
 
    ```bash
    git checkout <dev-branch>
    git pull upstream <dev-branch>
    ```
 
-3. Create a new topic branch (off the main project development branch) to contain your feature, change, or fix:
+2. Create a new topic branch (off the main project development branch) to contain your feature, change, or fix:
 
    ```bash
    git checkout -b <topic-branch-name>
    ```
 
-4. Install [pre-commit](https://pre-commit.com/) to ensure during development that each of your commits is properly formatted against our linter and formatters:
-
-   ```bash
-   pre-commit install
-   ```
-
-   > [!NOTE]\
-   > pre-commit is Python package that can be install via pip or any other Python package manager. You can also find it in our [requirements.txt](backend/requirements.txt) file.
-   >
-   > ```bash
-   > pip install pre-commit
-   > ```
-
```

Comment:
Ah - looks like this might've gotten left out when merging the upstream changes in `main`

Reply:
Oh, I'm fixing that now! It really was on merge :P